### PR TITLE
UHM-6902: Smoke Tests for Dispensing

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

**Review by mseaton — APPROVED** [review]
LGTM, thanks @mogoodrich !

File: src/main/java/org/openmrs/module/mirebalais/smoke/pageobjects/DispensingApp.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.presenceOfElementLocated;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class DispensingApp extends AbstractPageObject {
+
+    By activePrescriptionsSelector = By.xpath("//*[contains(text(), 'Active prescriptions')]");
+    By allPrescriptionsSelector = By.xpath("//*[contains(text(), 'All prescriptions')]");
```
Comment (cioan) [inline src/main/java/org/openmrs/module/mirebalais/smoke/pageobjects/DispensingApp.java:11]: We don't need to worry about translations b/c this smoke test is only for SL?

**Comment by mogoodrich** [comment]
Thanks @mseaton , merging!

**Review by cioan — APPROVED** [review]
Looks good, @mogoodrich !

**Pull request merged**